docs/guide/setup-a-webots-project-repository.md

```diff
@@ -0,0 +1,44 @@
+## Setup a Webots Project Repository
+
+### Requirements
```

Comment:
The webots.yaml file is only necessary on webots.cloud because we made it that way. However it is not necessary to be able to run a webots simulation on a similar platform that someone could create.